vcd_cli/user.py

```diff
@@ -19,6 +21,9 @@ def user(ctx):
 \b
         vcd user create 'my user' 'my password' 'role name'
            create user in the current organization with 'my user' 'my password' and 'role name' .
+\b
+        vcd user delete 'my user'
+           deletes user with username "my user" from the current organization. Will also delete vApps owned by the user. If user has running vApps, this command will result in error.
```

Comment:
I feel its better to move the side-effects of the command as a prompt when the command is executed.

Reply:
man/help pages often list side effects.  However I agree that we should prompt the user for confirmation along with the potential pitfall message.